src/stores/EndpointConfig/Hydrator.tsx

```diff
@@ -16,7 +17,7 @@ export const EndpointConfigHydrator = ({ children }: BaseComponentProps) => {
     const entityType = useEntityType();
     const workflow = useEntityWorkflow();
 
-    const [runHydration, setRunHydration] = useState(true);
+    const runHydration = useRef(true);
```

Comment:
This never needed to be state. Doubt it causes any issues - but better to be safe.